### ISSUE TITLE
Comments too dark

### ISSUE THREAD

**Issue by urielzen** [ghaschel/vscode-angular-html#16]
I find the comments to be too dark for me.

![image](https://user-images.githubusercontent.com/7947179/49092194-141ea880-f227-11e8-8b9a-a3631506ef36.png)

I understand that it is the dark-plus-syntax issue. So I submitted an issue in that repository
https://github.com/dunstontc/dark-plus-syntax/issues/2

I wanted to see what you think about changing the html comments in your extension so it is not dependent on the dark-plus-syntax?
Thanks

**Comment by ghaschel** [comment]
I'm afraid there isn't much I can do in this matter... 

I saw your issue there a few days ago, and I also agree, at least the text inside `<!-- -->` could be a little lighter. As far as the plugin goes, It only matches the comments and apply scopes names to that. Then is up for the theme to apply colors...

Currently the comments are matched like this:

`<!--` - punctuation.definition.comment.begin.html
`*anything inside* ` - comment.block.html
`-->` - punctuation.definition.comment.end.html

I tested with a different scope here, and it looks good, but I shouldn't do it, because i would end up breaking comments in other themes... eg: I tested with the same scope as `<` (tag opening) in dark-plus-syntax, but that would end making the comments red in some themes.
